checador/app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,41 @@
+class CompaniesController < ApplicationController
+   
+    def index
+        @companies = Company.all
+      end
+      
+      def new
+        @companies = Company.new
+      end
+    
+      def create
+        @companies = Company.new(company_params)
+        @companies.save
+        redirect_to companies_path
+      end
+    
+      def edit
+        set_company
+      end
+    
+      def update
+        set_company
```

Comment:
You can use before_action hook to call this method

Reply:
ok